community/kernel/src/main/java/org/neo4j/kernel/impl/api/StateHandlingStatementOperations.java

```diff
@@ -120,6 +119,22 @@ public StateHandlingStatementOperations(
     }
 
     // <Cursors>
+
+    @Override
+    public Cursor<NodeItem> loadNodeById( KernelStatement statement, long nodeId ) throws EntityNotFoundException
```

Comment:
We already have couple of methods for node and relationship cursors like `nodeCursor` and `relationshipCursor`. 
New methods does not follow common name convention